docs/Logging.md

```diff
@@ -0,0 +1,43 @@
+# Logging
+You can log a message to see in DevOps console. The library use the [Fastify logging system](https://www.fastify.io/docs/v2.0.x/Logging/), that is based on [pino](https://github.com/pinojs/pino).
+
+To log messages call these methods on the logger instance of request. Logging is enabled by default. Therefore you can call on `request.log`:
+* `debug()`
+* `info()`
+* `warn()`
+* `error()`
+* `fatal()`
+
+Each method creates a log with the homonym level.
+
```

Comment:
By default the library will generate two logs for each request, one representing the incoming request and one for the request completion, logs are created with trace and info levels respectively and already provide useful information for later analysis or debugging, if you need more you can add your own logs.